content/docs/develop/userscripts/_index.md

```diff
@@ -31,12 +31,12 @@ Each item of the array must have the following properties:
 Example manifest:
 ```json
 {
-  "name": "Scratch Messaging",
-  "description": "Provides easy reading and replying to your Scratch messages.",
+  "name": "Copy link to comment button",
+  "description": "Adds a \"Copy Link\" button to all comments on the website, next to the \"Report\" button.",
   "userscripts": [
     {
       "url": "userscript.js",
-      "matches": ["projects", "https://scratch.mit.edu/", "profiles"]
+      "matches": ["projects", "https://scratch.mit.edu/", "profiles", "studios"]
```

Comment:
I think it should follow the `addon.json` exactly, but I see why you want to include a full URL there.

Reply:
Yeah, it's a compromise.